test/scheduler_visualisation_test.py

```diff
@@ -292,7 +292,7 @@ def test_dep_graph_missing_deps(self):
 
         fail = dep_graph[BadReqTask(succeed=False).task_id]
         self.assertEqual(fail[u'name'], 'BadReqTask')
-        self.assertEqual(fail[u'params'], {'task_id': BadReqTask(succeed=False).task_id})
+        self.assertEqual(fail[u'params'], {'succeed': 'False'})
```

Comment:
I think it is good to have information about the passed parameters for UNKNOWN tasks in the dependency graph. However, if a task gets status UNKNOWN for reasons different from the ones managed in this PR, then the code from https://github.com/spotify/luigi/pull/1569 is most likely executed, and the _task_id_ would be displayed in the dependency graph instead of the afore-mentioned parameters: when this is the case, If we want a consistent behaviour I would suggest not to display anything else apart from the task name. Such case should be covered with a new test as well?